app/scripts/metamask-controller.js

```diff
@@ -3518,6 +3518,12 @@ export default class MetamaskController extends EventEmitter {
    * Locks MetaMask
    */
   setLocked() {
+    const [trezorKeyring] = this.keyringController.getKeyringsByType(
+      'Trezor Hardware',
```

Comment:
We should use `KEYRING_TYPES.TREZOR` from `shared/constants/hardware-wallets.js` here

Reply:
good call, added in latest commit